azure-pipelines.yml

```diff
@@ -42,4 +42,3 @@ jobs:
     - bash: |
         python -m pip install -r requirements-dev.txt
         python ./bin/run_tests.py
-
```

Comment:
Why? It's good to have a newline at the end of the code, no? We have that in all files, I thought?

Reply:
This is result of `end-of-file-fixer` I think that this is nice. Few tools expect last empty line (ex cat two files)

Reply:
> Few tools expect last empty line (ex cat two files)

Yes, exactly. That's why I was wondering why you are removing the last line?

Reply:
Oh, nevermind. There still is a final new line! I looked at the wrong file (`appveyor.yml` instead of `azure-pipelines.yml`). Sorry!